docs/source/examples/Notebook/rstversions/JavaScript Notebook Extensions.rst

```diff
@@ -80,29 +80,27 @@ in the behavior of the notebook.
 custom.js with others.
 
 Back to theory
-==============
+''''''''''''''
```

Comment:
Descending to the fifth level in Dante's documentation circles. There's not anything wrong with this line. Leaving this as a reminder to myself to add the project's RST levels to contributor docs ``( =, -, ~, ^, ''')`` since I recently cleaned up a mess of formatting on another project. :fire: